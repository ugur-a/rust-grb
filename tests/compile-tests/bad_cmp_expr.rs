use grb::prelude::*;
mod utils;

fn main() -> grb::Result<()> {
    create_model!(_g, m, x, y, z);
    c!(x + y == );
    c!(==);
    c!(x);
    c!(>=3);
    Ok(())
}
