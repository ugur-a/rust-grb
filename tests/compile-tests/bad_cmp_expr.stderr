error: unexpected end of input, expected expression
 --> tests/compile-tests/bad_cmp_expr.rs:6:5
  |
6 |     c!(x + y == );
  |     ^^^^^^^^^^^^^
  |
  = note: this error originates in the macro `c` (in Nightly builds, run with -Z macro-backtrace for more info)

error: expected expression
 --> tests/compile-tests/bad_cmp_expr.rs:7:8
  |
7 |     c!(==);
  |        ^

error: expected binary operation
 --> tests/compile-tests/bad_cmp_expr.rs:8:8
  |
8 |     c!(x);
  |        ^

error: expected expression
 --> tests/compile-tests/bad_cmp_expr.rs:9:8
  |
9 |     c!(>=3);
  |        ^
